src/gui/gui.cpp

```diff
@@ -35,9 +35,9 @@ static bool initialized = false;
 
 void init()
 {
-	if(initialized) {
-		return;
-	}
+//	if(initialized) {
```

Comment:
These should be deleted rather than commented out.

Reply:
`initialized` has been completely removed since multiple initializations are now possible.